app/models/submission.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Submission < ApplicationRecord
+  # Associations
+  belongs_to :submitter, class_name: 'User'
+  belongs_to :challenge
+  belongs_to :phase
+  belongs_to :manager, class_name: 'User'
+
+  # Fields
+  attribute :title, :string
+  attribute :brief_description, :string
+  attribute :brief_description_delta, :string
```

Comment:
I don't think we need these _delta and _length fields, so I'd rather not bring extra noise on this model.
```suggestion
```